SB2018/2018-01-31.md

```diff
@@ -0,0 +1,26 @@
+---
+to: All Teams
+subject: Tech day reminder - Saturday 3rd Feb
+---
+
+Hi all,
+
+Just a reminder and some information on the tech day we're running on Saturday (3rd feb)!
+
+Tech days are a great opportunity for you to get robotics advice and help with technical issues from our volunteers. We may also have a tour of some university facilities as well.
+
+## Location
+The tech day will be held in [Building 25](http://data.southampton.ac.uk/building/25.html). There's a campus map with parking labelled attached. 
```

Comment:
"map with parking labelled attached" doesn't read well, maybe "map attached which shows the recommended parking location"

Reply:
Yea, i didn't really like that when writing it, but that's much nicer! Will update